apps/explorer-e2e/src/support/pages/transactions-page.js

```diff
@@ -0,0 +1,37 @@
+import BasePage from './base-page';
+
+export default class TransactionsPage extends BasePage {
+  blockRow = 'block-row';
+  blockHeight = 'block-height';
+  numberOfTransactions = 'num-txs';
+  validatorLink = 'validator-link';
+  blockTime = 'block-time';
+  refreshBtn = 'refresh';
+
+  validateTransactionsPagedisplayed() {
+    cy.getByTestId(this.blockRow).should('have.length.above', 1);
+    cy.getByTestId(this.blockHeight).first().should('not.be.empty');
+    cy.getByTestId(this.numberOfTransactions).first().should('not.be.empty');
+    cy.getByTestId(this.validatorLink).first().should('not.be.empty');
+    cy.getByTestId(this.blockTime).first().should('not.be.empty');
+  }
+
+  validateRefreshBtn() {
+    cy.getByTestId(this.blockHeight)
+      .first()
+      .invoke('text')
+      .then((blockHeightTxt) => {
+        cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
anything we can do to make this more implicit ?

Reply:
Yeah I'd like to avoid the wait if possible. I can't think of a much better way to do this. Possibly putting it in a loop to keep clicking the refresh button until the block has changed but I think that's overkill for this.